modules/odc-creating-apiserversource.adoc

```diff
@@ -16,6 +16,8 @@ image::apiserversource-odc.png[Create an ApiServerSource]
 // .. Select **Resource** as the **Mode**. **Mode** is the mode that the receive adapter controller runs in. `Ref` sends only the reference to the resource. `Resource` sends the full resource.
 // TODO: clarify what this is used for. Out of scope for this PR since not required.
 .. Select the **Service Account Name** for the service account that you created.
+.. Select the **Sink** for the event source. A **Sink** can be either a **Resource**, such as a channel, broker, or service, or a **URI**.
```

Comment:
I think the later ones don't need to be bold, since this is generally just used in OCP docs where it's referring to something literal in the console. **Resource** here should probably be just Resource, same for URI?